.editorconfig

```diff
@@ -0,0 +1,15 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+[{*.md,*.yml}]
```

Comment:
Let's keep this at `[*.md]` instead as there are currently no YAML files.

Reply:
File has been updated, and I totally hear you on coding standards — I'm usually the person setting up CI pipelines on my team to run tests, enforce standards, and perform static code analysis 😄. I've updated the PR to more-closely follow what you have in place.